canonical-web-and-design/snapcraft.io
# Setup listing React app

## Done
Loaded a React application on the listing page if listing data is available and the feature flag is present

## QA
- Go to https://snapcraft-io-4034.demos.haus/notion-snap/listing?show_react_listing=true
- Check that there is a heading of "Snap listing page"
- [Reload the page without the query string](https://snapcraft-io-4034.demos.haus/notion-snap/listing) and check that the page looks and works as it currently does in production

## Issue
Fixes https://github.com/canonical-web-and-design/marketplace-tribe/issues/2640
